spec/see_dishes_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/models/dish.rb'
+
+
+describe Dish do
+  before do
+    subject = Dish.new
+    subject.name = 'Meatballs'
+    subject.price = 8.99
+    subject.save
+  end
+
+  it "is expected to have a name" do
+    expect(subject.first.name).to eq 'Meatballs'
```

Comment:
This used to be `Dish.first.name`. Did that not work? With that code, we are testing that a record was saved to the database.